src/microbe_stage/CompoundCloudSystem.cs

```diff
@@ -332,33 +330,25 @@ public void GetAllAvailableAt(Vector3 worldPosition, Dictionary<Compound, float>
             // relative origin point is calculated and that is restricted by
             // checking if the point is within the circle before grabbing
 
-            // And apparently there isn't an algorithm to generate the points with closest ones first? so
-            // this goes through everything and keeps the closest found point
-
-            int xEnd = (int)Mathf.Round(cloudRelativeX + localRadius);
-            int yEnd = (int)Mathf.Round(cloudRelativeY + localRadius);
+            // TODO: pull these to a constant to not constantly recalculate them
```

Comment:
TODO comment spotted

Reply:
Is there anywhere else you want to move this to? It turned out to be unneeded for performance, so I'd just drop the TODO and leave it as is.

Reply:
Constants or MathUtils.